runtime/calamari/src/lib.rs

```diff
@@ -114,7 +114,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_version: 3110,
 	impl_version: 1,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 2,
+	transaction_version: 3,
```

Comment:
I bumped `transaction_version` because the count of extrinsics from collator-selection has changed.
I added new extrinsics.